assets/js/googlesitekit/datastore/user/dismissed-items.js

```diff
@@ -154,6 +166,28 @@ const baseActions = {
 			);
 		}
 	),
+	setItemDimissingState( slug, state ) {
+		return {
+			payload: { slug, state },
+			type: 'SET_ITEM_DISMISSING_STATE',
+		};
+	},
+};
+
+const baseReducer = ( state, { type, payload } ) => {
+	switch ( type ) {
+		case 'SET_ITEM_DISMISSING_STATE':
+			const { slug, state: isDismissing } = payload;
+			return {
+				...state,
+				isDismissingItems: {
+					[ stringifyObject( slug ) ]: isDismissing,
```

Comment:
Evan actually mentioned in comments on the IB that the slugs are unique so we don't have to use `stringifyObject` here we can just set the is dismissing state on for the slug as passed.